source/manual/2nd-line.html.md

```diff
@@ -0,0 +1,70 @@
+---
+owner_slack: "#govuk-2ndline"
+title: 2nd line
+parent: "/manual.html"
+layout: manual_layout
+section: 2nd line
+type: learn
+last_reviewed_on: 2019-08-16
+review_in: 3 months
+---
+
+2nd line has three main reponsibilities:
+
+- Monitoring the state of the GOV.UK infrastructure
+- Investigating and responding to technical bug reports
+- Providing first line support to queries from data.gov.uk users
+
+If you're new to 2nd line, read about our [working patterns, ceremonies and policies](/manual/welcome-to-2nd-line.html).
+
+## Monitoring
+
+We have a [2nd line dashboard][2nd-line-dashboard] showing a high level overview of the state of
+the GOV.UK environments. You can also [install our Chrome extension][chrome-extension] if you want
+a permanently visible overview. You will need to be on the [VPN](/manual/vpn.html) if accessing from home.
+
+### Icinga
+
+We use Icinga to monitor our platform and alert us when things go wrong. Many alerts have corresponding
+documentation in these developer docs, detailing how to respond.
+
+Record critical alerts that aren't easily solved to the [GOV.UK 2nd line Trello board][2nd-line-trello-board]
+to help inform the Platform Health and the GOV.UK AWS migration team. 2nd line should investigate these alerts
+when there is downtime - you do not necessarily have to fix them.
+
+[Read more about Icinga](/manual/icinga.html).
+
+### PagerDuty
+
+Some alerts are urgent enough to warrant immediate attention, such as parts of the site becoming
+unavailable or large quantities of error pages being served. We use [PagerDuty](https://governmentdigitalservice.pagerduty.com)
+to escalate these alerts to the primary and secondary engineers on 2nd line during office hours.
```

Comment:
Maybe say "to notify", rather than "to escalate", as I'm not sure what escalating means in this context?